video/agon.h

```diff
@@ -26,7 +26,7 @@
 #define UART_RTS				13		// The ESP32 RTS pin (eZ80 CTS)
 #define UART_CTS	 			14		// The ESP32 CTS pin (eZ80 RTS)
 
-#define COMMS_TIMEOUT			10		// Timeout for VDP commands (ms)
+#define COMMS_TIMEOUT			100		// Timeout for VDP commands (ms)

```

Comment:
the shorter timeout here was experimental...  the code to handle incoming buffered commands used `readByte_t` for a while and in a BBC BASIC test program that sent over sound samples using buffered commands this proved problematic, as occasionally a byte would time out, throwing off the transfer, and resulting in some sample data being interpreted as VDU commands

the underlying cause of the timeout was most likely delays on the z80 side reading data from the SD card.

the desire to use the `readByte_t` variant is to try to avoid having the VDP get stuck in a blocked "waiting for data" state.  this is still desirable behaviour and should be addressed